### PR TITLE
Update Frisby to an ES6 class

### PR DESCRIPTION
**Pull request by paulmelnikow**
- Apply and enforce consistent indentation
- Switch a couple JSDoc comments to consistent format
- Consistent spacing between methods

### CODE REVIEW

File: lib/icedfrisby.js [hunk 1]
```diff
@@ -125,324 +127,322 @@ function _useAppImpl (app, basePath) {
...
-  return this;
-};
+  /**
+   * @param {object} headers - header object {k;v, k:v}
+   * @return {object}
+   * @desc Add group of HTTP headers together
+   */
+  addHeaders (headers) {
+    const self = this;
+    _.forEach(headers, function(val, key) {
```
Comment (MarkHerhold) [inline lib/icedfrisby.js]: Just a thought, but would it make sense to start using arrow functions for this upgrade as well to eliminate some of the `const self = this` statements?

Comment (paulmelnikow) [inline lib/icedfrisby.js]: Yes, that would be great! They'll make these things much more readable.

Since those changes are easy to get wrong, having more eyes to review them is good. It's impossible to review the diff here, so I would tend to make those in a separate commit.

Comment (paulmelnikow) [inline lib/icedfrisby.js]: This diff is more contextual than I expected. Though I'd still feel more comfortable merging this first, and then tackling arrow functions.

**Comment by MarkHerhold** [comment]
Fine by me! Do whatever you like, I was just making a passing suggestion.
